Makefile

```diff
@@ -74,8 +73,17 @@ help: ## Display this help.
 
 ##@ Development
 
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+manifests: controller-gen yq ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+	$(CONTROLLER_GEN) crd:crdVersions=v1 rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(YQ) -i eval 'del(.metadata.creationTimestamp)' config/crd/bases/infrastructure.cluster.x-k8s.io_bootstrapkubeconfigs.yaml
```

Comment:
Can we use wildcard to merge multiple commands into one. 
E.g. `$(YQ) -i eval 'del(.metadata.creationTimestamp)' config/crd/bases/*`

Reply:
I tried this and it doesn't work. The argument `-i` only takes into account the first file.
`-i, --inplace               update the yaml file inplace of first yaml file given.`

Reply:
Got it. Thanks for checking.